test/bin/scenario.sh

```diff
@@ -35,16 +80,14 @@ sos_report() {
             # skip log files, etc.
             continue
         fi
-        ip=$(cat "${vmdir}/ip")
-        if [ -z "${ip}" ] ; then
+        local -r vmname=$(basename "${vmdir}")
+        local -r ip=$(get_vm_property "${vmname}" ip)
+        if [ -z "${ip}" ]; then
             # skip hosts without NICs
             # FIXME: use virsh to copy sos report files
             continue
         fi
-        # Copy the sos helper for compatibility, it is only available in 4.14 RPMs
-        scp "${ROOTDIR}/scripts/microshift-sos-report.sh" "redhat@${ip}":/tmp
-        ssh "redhat@${ip}" \
-            "[ -f /usr/bin/microshift-sos-report ] && sudo /usr/bin/microshift-sos-report || sudo /tmp/microshift-sos-report.sh ; sudo chmod +r /tmp/sosreport*"
+        run_command_on_vm "${vmname}" "sudo sos report --quiet --batch --all-logs --tmp-dir /tmp && sudo chmod +r /tmp/sosreport*"
```

Comment:
I'm not sure we need to go back to collecting all the logs. They slow down the process and they contain too much unnecessary information. I'd rather continue using the `microshift-sos-report` script.

Reply:
Note: I just approved a fix in #2565 which fixes the logic of sos collection when MicroShift is not installed.